spec/amp-var-substitutions.md

```diff
@@ -96,6 +96,115 @@ For instance:
 ```
 may make a request to something like `https://foo.com/pixel?href=https%3A%2F%2Fpinterest.com%2F`.
 
+### DOM_EXT_IMG_COUNT
```

Comment:
/cc @avimehta what's your advise on where to keep this documentation? Do we normally combine it all together in one doc or keep it within the extension itself and just link it out?